### PR TITLE
Natalia Buzila WDPT RMT 5APR

### PR DESCRIPTION
**Pull request by nataliabuzila**
Iterations 1-4, 5 incomplete; styling incomplete

### CODE REVIEW

File: src/components/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+function Carousel ({images}) {
+
+    const [carouselNr, setCarouselNr] = useState(0)
+
+    function left () {
+        if(carouselNr>0) 
+        setCarouselNr((oldCarouselNr)=>oldCarouselNr-1)
+        else
+        setCarouselNr((oldCarouselNr)=>3)
```
Comment (matfire) [inline src/components/Carousel.js:11]: why 3?

Comment (Matteo-Ironhack) [inline src/components/Carousel.js:11]: why 3?

File: src/components/FaceBook.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import profiles from '../data/berlin.json';
```
Comment (Matteo-Ironhack) [inline src/components/FaceBook.js:1]: it is usually better to save the content of a file in a state variable

File: src/components/SignupPage.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    return (
+        <div>
+            <h3>Iteration 15</h3>
+            <form onSubmit={handleSubmit}>
+
+                <label>Email</label>
+                <input type='text' name='email' value={email} onChange={handleEmail}/>
+
+                <label>Password</label>
+                <input type='text' name='password' value={password} onChange={handlePassword} />
```
Comment (Matteo-Ironhack) [inline src/components/SignupPage.js:26]: to obfuscate the password value you can use a type `password`

File: src/components/SignupPage.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+                <label>Nationality</label>
+                <select value = {nationality} onChange={handleNationality} >
+                    <option value="en">English</option>
+                    <option value="de">German</option>
+                    <option value="fr">French</option>
+                </select>
+
+                <button type="submit">Sign up</button>
+            </form>
```
Comment (Matteo-Ironhack) [inline src/components/SignupPage.js:36]: you are missing a step; this task also needed to have some information displayed beneath the form

**Pull request closed without merging**